pkg/pillar/types/edgeviewtypes.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021-2022 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Types which feed in and out of the verifier
+
+package types
+
+import "time"
+
+// Types for edge-view on device
+
+const (
```

Comment:
Is it easier to have separate files in /run/edgeview as opposed to a single fil with all the settings?
Could there be races if edge-view reacts to changes one some but not all files have been written?
With a single file you can use the atomic WriteRename()